airbyte-webapp/src/views/Connection/CatalogTree/CatalogTreeSubheader.module.scss

```diff
@@ -0,0 +1,19 @@
+%catalogHeader {
```

Comment:
This is repeated between the CatalogTreeHeader and the CatalogTreeSubheader module. I'm not sure how to reuse this with SCSS modules.

Reply:
It looks like mixins may be favored over placeholders for modules: https://github.com/sass/dart-sass/issues/1042